paz/processors/pose.py

```diff
@@ -127,3 +132,249 @@ def call(self, boxes2D):
             y_center = (z_center * v) / self.focal_length
             hands_center.append([x_center, y_center, z_center])
         return np.array(hands_center)
+
+
+class ComputeCameraParameter(Processor):
+    """Computes camera parameter given camera matrix
+    and scale normalization factor of translation.
+
+    # Arguments
+        camera_matrix: Array of shape `(3, 3)` camera matrix.
+        translation_scale_norm: Float, factor to change units.
+            EfficientPose internally works with meter and if the
+            dataset unit is mm for example, then this parameter
+            should be set to 1000.
+    """
+    def __init__(self, camera_matrix, translation_scale_norm):
+        self.camera_matrix = camera_matrix
+        self.translation_scale_norm = translation_scale_norm
+        super(ComputeCameraParameter, self).__init__()
+
+    def call(self, image_scale):
+        return np.array([self.camera_matrix[0, 0], self.camera_matrix[1, 1],
+                         self.camera_matrix[0, 2], self.camera_matrix[1, 2],
+                         self.translation_scale_norm, image_scale])
+
+
+class MatchPoses(Processor):
+    """Match prior boxes with ground truth poses.
+
+    # Arguments
+        prior_boxes: Numpy array of shape (num_boxes, 4).
+        iou: Float in [0, 1]. Intersection over union in which prior
+            boxes will be considered positive. A positive box is box
+            with a class different than `background`.
+    """
+    def __init__(self, prior_boxes, iou=.5):
+        self.prior_boxes = prior_boxes
+        self.iou = iou
+        super(MatchPoses, self).__init__()
+
+    def call(self, boxes, poses):
+        return match_poses(boxes, poses, self.prior_boxes, self.iou)
+
+
+class TransformRotation(Processor):
+    """Computes axis angle rotation vector from a rotation matrix.
```

Comment:
The name does not seem to correspond to the documentation? The name seems also very vague.